### PR TITLE
DQE check rerun functionality

### PR DESCRIPTION
**Pull request by svetsolo**
Implementing rerun functionality based on checkid input. Extending results functionality to show manual versus scheduled checks.

### CODE REVIEW

File: code/processes/dqe.q [hunk 1]
```diff
@@ -44,35 +44,35 @@ fillprocname:{[rs;h]
   (flip a),val,'`
   }
 
-dupchk:{[idnum;proc]                                                                                            /- checks for unfinished runs that match the new run
+dupchk:{[man;idnum;proc]                                                                                            /- checks for unfinished runs that match the new run
```
Comment (ThomasSmyth) [inline code/processes/dqe.q]: Realign all comments that have moved due to the extra argument.

File: code/processes/dqe.q [hunk 2]
```diff
@@ -44,35 +44,35 @@ fillprocname:{[rs;h]
...
-  `.dqe.results set update chkstatus:`failed,output:0b,descp:c#enlist error from .dqe.results where id=idnum, procschk=proc,chkstatus=`started;
+  `.dqe.results set update chkstatus:`failed,output:0b,descp:c#enlist error,manual:man from .dqe.results where id=idnum, procschk=proc,chkstatus=`started;
   }
 
-postback:{[idnum;proc;result]                                                                                   /- function that updates the results table with the check result
+postback:{[man;idnum;proc;result]                                                                                   /- function that updates the results table with the check result
   $["e"=first result;                                                                                           /- checks if error returned from server side
-  .dqe.failchk[idnum;result;proc];
-  `.dqe.results set update endtime:.z.p,output:first result,descp:enlist last result,chkstatus:`complete from .dqe.results where id=idnum,procschk=proc,chkstatus=`started];
+  .dqe.failchk[man;idnum;result;proc];
```
Comment (ThomasSmyth) [inline code/processes/dqe.q]: Now that you're editing this line anyway can you fix the indent?

Comment (svetsolo) [reply code/processes/dqe.q]: Fixed it , thanks

File: code/processes/dqe.q [hunk 3]
```diff
@@ -98,4 +105,9 @@ update endtime:?[0W=endtime;0Wp;.z.d+endtime] from `.dqe.configtable
 / Sample runcheck:
 / show .dqe.results
 / Load up timers
+
+/Sample reruncheck
+/chkid:3
+/.dqe.reruncheck[chkid]
```
Comment (ThomasSmyth) [inline code/processes/dqe.q]: Remove this comment.

**Pull request merged**